src/themes/default/components/core/blocks/Auth/Login.vue

```diff
@@ -186,7 +180,5 @@ export default {
 
   .validation-error {
     display: block;
```

Comment:
add `display: block` to positioning.scss

Reply:
Done